docs/cryptodoc/src/05_06_spx.rst

```diff
@@ -298,6 +298,7 @@ within the ``SphincsPlus_PrivateKey`` constructor. It works as follows:
    **Input:**
 
    -  ``rng``: random number generator
+   -  ``params``: Sphincs+ parameters
 
```

Comment:
For consistency I would write:
SPHINCS\ :sup:'+'
or if it does not work in the admonition role:
SPHINCS+